src/gui/PasswordGeneratorWidget.cpp

```diff
@@ -196,13 +248,13 @@ void PasswordGeneratorWidget::colorStrengthIndicator(double entropy)
     // Set the color and background based on entropy
     // colors are taking from the KDE breeze palette
     // <https://community.kde.org/KDE_Visual_Design_Group/HIG/Color>
-    if (entropy < 35) {
+    if (entropy < 40) {
```

Comment:
@TheZ3ro maybe these thresholds could be configurable at some point, for users that have different security requirements.